### ISSUE TITLE
Additional installation instructions in readme.md

### ISSUE THREAD

**Issue by jonphipps** [UserScape/Laravel-Bundles#42]
It would be good to remove the existing key from application/config/application.php and provide explicit instructions on how to generate a key from artisan, instead of shipping with an existing key (which may raise a security issue?)

An explicit instruction to make sure that the entire 'storage' directory is writable would be helpful. It may be entirely self-evident, but it bit me as someone completely clueless about the laravel folder structure.

**Comment by ericlbarnes** [comment]
Yeah good ideas. The application key is actually not the one we have in production as we are using laravel environments for that.  I will see if I can't get all this updated.
